bundle/edu.gemini.spModel.io/src/test/scala/edu/gemini/spModel/io/impl/migration/to2015B/TemplateTargetConversionTest.scala

```diff
@@ -0,0 +1,98 @@
+package edu.gemini.spModel.io.impl.migration.to2015B
+
+import edu.gemini.spModel.obscomp.SPNote
+import edu.gemini.spModel.target.system.{NonSiderealTarget, HmsDegTarget}
+
+import java.io.{StringReader, StringWriter, InputStreamReader}
+
+import edu.gemini.pot.sp.{ISPTemplateFolder, ISPTemplateGroup, ISPProgram}
+import edu.gemini.pot.spdb.{IDBDatabaseService, DBLocalDatabase}
+import edu.gemini.spModel.io.impl.{PioSpXmlWriter, PioSpXmlParser}
+import edu.gemini.spModel.template.{TemplateParameters, TemplateGroup}
+
+import org.junit.Test
+import org.junit.Assert._
+
+import scala.collection.JavaConverters._
+
+// a rudimentary test to make sure it doesn't blow up
+
+class TemplateTargetConversionTest {
+  private def withTestOdb(block: IDBDatabaseService => Unit): Unit = {
```

Comment:
I bet we are doing this in many places on the codebase, do you think it should go to a common testing package?